src/components/AddProfileForm.css

```diff
@@ -0,0 +1,61 @@
+.Form-Wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.ProfileForm {
+  color: #cc8696;
+  margin: 1px;
+  align-items: center;
+}
+.FormButtons {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.Upload {
+  display: flex;
+  flex-direction: column;
+  padding-top: 15px;
+  color: #cc8696;
+
+  margin-top: 20px;
+}
+
+.Submit {
+  height: 30px;
+  width: 45px;
+  border-radius: 10px;
+  text-align: center;
+  transition: 1s;
+  background-color: lightgrey;
+  color: whitesmoke;
+}
+.Cancel {
+  height: 30px;
+  width: 45px;
+  border-radius: 10px;
+  text-align: center;
+  transition: 1s;
+  background-color: lightgrey;
+  color: whitesmoke;
+}
+#userName {
+  width: 300px;
+  padding: 10px;
+}
+
+#userBio {
+  height: 200px;
+  width: 300px;
+  overflow-y: scroll;
+  padding: 10px;
+}
+.ptag {
+  letter-spacing: 4px;
+  padding-top: 20px;
+  padding-bottom: 20px;
```

Comment:
padding in this context can sometimes cause weird unforseen click areas. 
For example if you give your p-tags a background of any color, there won't be any space between them. 
You might be better off with using margin instead to have some  actual distance between the items.

Reply:
I'll check it.